### PR TITLE
Release 3.3.4

### PR DESCRIPTION
**Pull request by FinalAngel**
- update to bootstrap 3.3.5
- update dependencies
- simplified templates
- changed $speed value to $speed-base

### CODE REVIEW

**Comment by vxsx** [comment]
LGTM

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,6 +3,14 @@ Aldryn Boilerplate Bootstrap 3
 ##############################
 
 
+3.3.5
```
Comment (vxsx) [inline CHANGELOG.rst]: 3.3.4 :D

Comment (FinalAngel) [inline CHANGELOG.rst]: fixed :D

**Comment by Flight** [comment]
LGTM

**Comment by Brankub** [comment]
LGTM

**Comment by lory87** [comment]
:+1:

File: templates/fullwidth.html [hunk 2]
```diff
@@ -3,10 +3,6 @@
 
 {% block body_class %}tpl-fullwidth{% endblock %}
 
-{% block extend_base %}
-<div class="container">
```
Comment (vxsx) [inline templates/fullwidth.html]: @FinalAngel why did you remove container everywhere?

Comment (FinalAngel) [inline templates/fullwidth.html]: The container is now added within `base.html`

Comment (vxsx) [inline templates/fullwidth.html]: not that i'm against it, it's a correct move, but this is potentially a breaking change for users that didn't touch the fullwidth.html in their aldryn projects and relied on the container to be there

Comment (FinalAngel) [inline templates/fullwidth.html]: AFAIK only protected files get overridden on an update currently. It has been tested beforehand on the xplorer theme
